config/main.py

```diff
@@ -435,15 +447,25 @@ def del_vlan_member(ctx, vid, interface_name):
     vlan = db.get_entry('VLAN', vlan_name)
 
     if get_interface_mode() == "alias": 
-        interface_name = interface_alias(interface_name)
+        default_name = interface_alias_to_name(interface_name)
```

Comment:
Please refactor this code in the same manner as I suggested in my comment on line 411.